cs/src/Management/TunnelManagementClient.cs

```diff
@@ -519,6 +519,15 @@ private string UserLimitsPath
 
             request.Headers.UserAgent.Add(TunnelSdkUserAgent);
 
+            const string regKeyPath = @"Software\Policies\Microsoft\Tunnels";
+            var defaultOnError = true;
+            RegistryTools policies = new RegistryTools();
+            var keyValue = (string)policies.GetRegistryValueFromLocalMachineRoot(regKeyPath, defaultOnError);
+            if (keyValue != null)
+            {
+                request.Headers.Add("Policies", keyValue);

```

Comment:
How about `User-Agent-Policies` header name if it's not decided yet?

Also can we limit the length of `keyValue` to some reasonable length? Our ingress has max 64K header length, but that's for all of the headers. So, how about limiting this header to 1K?

Reply:
In this case should Policies be added as to the existing User-Agent property?

Reply:
Having policies in the User-Agent header is ok. There would be more work to parse it from there on the server side or in telemetry queries. Having policies in its own header is also ok. Less work for the service and telemetry queries to parse it. I'm just thinking that we already have policies. So, to not confuse them with these policies, let's name the header differently, e.g. `User-Agent-Policies`.